examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2105,623 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
+		defer cursor.Close(ctx)
+		requireCursorLength(t, cursor, 4)
+	}
+	{
+		// Start Aggregation Example 5
+		pipeline := mongo.Pipeline{
+			{
+				{"$lookup", bson.D{
+					{"from", "air_airlines"},
+					{"let", bson.D{
+						{"constituents", "$airlines"}},
+					},
+					{"pipeline", bson.A{bson.D{
+						{"$match", bson.D{
+							{"$expr", bson.D{
+								{"$in", bson.A{"$name", "$$constituents"}},
+							}},
+						}},
+					}}},
+					{"as", "airlines"},
+				}},
+			},
+			{
+				{"$project", bson.D{
+					{"_id", 0},
+					{"name", 1},
+					{"airlines", bson.D{
+						{"$filter", bson.D{
+							{"input", "$airlines"},
+							{"as", "airline"},
+							{"cond", bson.D{
+								{"$eq", bson.A{"$$airline.country", "Canada"}},
+							}},
+						}},
+					}},
+				}},
+			},
+		}
+
+		cursor, err := airAlliancesColl.Aggregate(ctx, pipeline)
+
+		// End Aggregation Example 5
+
+		require.NoError(t, err)
+		defer cursor.Close(ctx)
+		requireCursorLength(t, cursor, 3)
+	}
+}
+
+// RunCommandExamples contains examples of RunCommand operations.
+func RunCommandExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	coll := db.Collection("restaurants")
+
+	err := coll.Drop(ctx)
+	require.NoError(t, err)
+
+	{
+		// Start RunCommand Example 1
+		restaurants := []interface{}{
+			bson.D{
+				{"name", "Chez Panisse"},
+				{"city", "Oakland"},
+				{"state", "California"},
+				{"country", "United States"},
+				{"rating", 4.4},
+			},
+			bson.D{
+				{"name", "Central"},
+				{"city", "Lima"},
+				{"country", "Peru"},
+				{"rating", 4.8},
+			},
+			bson.D{
+				{"name", "Eleven Madison Park"},
+				{"city", "New York City"},
+				{"state", "New York"},
+				{"country", "United States"},
+				{"rating", 4.6},
+			},
+			bson.D{
+				{"name", "Gaggan"},
+				{"city", "Bangkok"},
+				{"country", "Thailand"},
+				{"rating", 4.3},
+			},
+			bson.D{
+				{"name", "Dad's Grill"},
+				{"city", "Oklahoma City"},
+				{"state", "Oklahoma"},
+				{"country", "United States"},
+				{"rating", 2.1},
+			},
+		}
+
+		result, err := coll.InsertMany(ctx, restaurants)
+
+		// End RunCommand Example 1
+
+		require.NoError(t, err)
+		require.Len(t, result.InsertedIDs, 5)
+	}
+	{
+		// Start RunCommand Example 2
+		res := db.RunCommand(ctx, bson.D{{"buildInfo", 1}})
+
+		// End RunCommand Example 2
+
+		err := res.Err()
+		require.NoError(t, err)
+	}
+	{
+		// Start RunCommand Example 3
+		res := db.RunCommand(ctx, bson.D{{"collStats", "restaurants"}})
+
+		// End RunCommand Example 3
+
+		err := res.Err()
+		require.NoError(t, err)
+	}
+}
+
+// IndexExamples contains examples of Index operations.
+func IndexExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	recordsColl := db.Collection("records")
+	restaurantsColl := db.Collection("restaurants")
+
+	err := recordsColl.Drop(ctx)
+	require.NoError(t, err)
+	err = restaurantsColl.Drop(ctx)
+	require.NoError(t, err)
+
+	{
+		// Start Index Example 1
```

Comment:
same as above, populating the collections isn't part of an example

Reply:
See comment above, and lmk what you think 👍